src/components/DocSidebar.tsx

```diff
@@ -40,15 +51,23 @@ const DocSidebar: FunctionalComponent<{ headers: any[]; editHref: string }> = ({
   return (
     <nav>
       <div>
-        <h2 class="heading">Contents</h2>
+        <h2 className="heading">Contents</h2>
```

Comment:
Preact actually supports `class` if you prefer that!

Reply:
Oh! My editor was yelling at me, I guess it thought the file was vanilla React then.